libraries/eventformatter/api/src/main/kotlin/io/element/android/libraries/eventformatter/api/TimelineEventFormatter.kt

```diff
@@ -18,6 +18,6 @@ package io.element.android.libraries.eventformatter.api
 
 import io.element.android.libraries.matrix.api.timeline.item.event.EventTimelineItem
 
-interface TimelineEventFormatter {
+fun interface TimelineEventFormatter {
```

Comment:
For this one, eventually you could have simplified [this code](https://github.com/vector-im/element-x-android/blob/develop/features/messages/impl/src/test/kotlin/io/element/android/features/messages/fixtures/timelineItemsFactory.kt#L75).

But it's probably better to update it and write a class like:

```kotlin
class NoOpTimelineEventFormatter() : TimelineEventFormatter {
    override fun format(event: EventTimelineItem): CharSequence = ""
}
```

instead.